cmd/exec/operations.go

```diff
@@ -666,16 +666,9 @@ func executeCreateCloudEventFilter(filterName string, data io.Reader, force bool
 
 	}
 
-	if resp.StatusCode == http.StatusConflict {
```

Comment:
Why did you change this from being based on the http response status code?

Reply:
statusConflict means a filter wants to be creating which already exist. So if this appeared it shouldn't be updated only by forcing.